tests/bivariate/test_copula_selection.py

```diff
@@ -20,6 +20,7 @@ def test_snippet(self):
         c0 = Copula('clayton')
         c0.fit(U, V)
         result = c0.get_cdf()([0, 0.1, 0.2], [0, 0.1, 0.8])
+        print(result)
```

Comment:
Please, delete this line, statements used for developing should never be committed ( `print`s, `ipdb`s, etc, etc). If you intend to output a message to the user, which is not the case here, use a logger.